src/components/ebay-tabs/marko-tag.json

```diff
@@ -6,8 +6,7 @@
     "type": "expression"
   },
   "@html-attributes": "expression",
-  "@index": "string",
-  "@fake": "boolean",
+  "@index": "number",
```

Comment:
Not part of the scope of your changes I know, but I find `index` a little cryptic...

Reply:
Changed it to `current` since that's what we use in other places.

Reply:
`currrentIndex` maybe? We use `current` as an asbtraction of `aria-current`.

Reply:
Actually, how about `selectedIndex`? Ties in with aria-selected for the tab role:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role